docker/Dockerfile.debug

```diff
@@ -15,6 +15,7 @@ RUN apt-get update && \
     emacs \
     wrk \
     lsof \
+    busybox \
```

Comment:
If we have emacs - I don't think we need to worry too much about the few utilities that busybox is duplicating or adds :-) Added it for 'httpd' util, helps in some tests, and traceroute. 

We should probably have a separate thread about cleaning up the list - IMHO emacs is a bit too much, and wrk and few others could be added in separate images to load aside, don't need to be in the proxy.